src/upgrade/upgrade.window.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * @copyright Copyright (c) 2023 Maksim Sukharev <antreesy.web@gmail.com
+ *
+ * @author Grigorii Shartsev <grigorii.shartsev@nextcloud.com>
+ * @author Maksim Sukharev <antreesy.web@gmail.com>
```

Comment:
**typo:** missed `>` and unnecessaries `@author` tags

```suggestion
/*
 * @copyright Copyright (c) 2023 Maksim Sukharev <antreesy.web@gmail.com>
```

And the same in other new files